src/tinystr16.rs

```diff
@@ -25,6 +25,11 @@ use crate::Error;
 pub struct TinyStr16(NonZeroU128);
 
 impl TinyStr16 {
+    #[inline(always)]
+    pub fn from_utf8(v: &[u8]) -> Result<Self, Error> {
```

Comment:
Why calling it `from_utf8`? Isn't the point that this may be invalid utf-8? I'd call it `from_bytes` or such, probably.